README.md

```diff
@@ -140,6 +140,7 @@ For large releases and/or if there are deprecations or nml syntax changes, provi
 Example: https://github.com/OpenTTD/nml/blob/master/docs/changelog.txt
 4. Publish a new release using the release tool in the GitHub project: https://github.com/OpenTTD/nml/releases/new
 5. GitHub Actions will build the release and publish to PyPI (the Python package index).
+Download the source tarball, then upload it to the GitHub release (helpful for debian package maintainers).
```

Comment:
Why not let the workflow do this directly, just like the win64 build is already uploaded as an artifact?

Reply:
See the step below - it's not currently done by the github workflow ;)

Reply:
Doing so is on the todo list though...

Reply:
We could probably merge #108 and this PR together

Reply:
Oh, seems this needs #108 to really directly upload, but I guess that would be even nicer?